RFC.md

```diff
@@ -79,21 +82,27 @@ sequenceDiagram
 
 ### 4.1 Tool Definition Schema
 
+[//]: # (We need to separate the request and the repsonse parts of the tool definition schema.)
+[//]: # (Would expect a POST requests against /tools with some payload to create a new tool, and a GET request against /tools/{tool_id} to get the tool definition.)
+
 The Tool Definition Schema establishes the properties and required fields to describe a tool. It consists of the following sections:
 
 - **Metadata:**
 
   - **`$schema`**: URI defining the JSON Schema version.
+  - **`id`**: A unique identifier for the tool.
   - **`name`**: A human-readable name for the tool.
-  - **`fully_qualified_name`**: A unique identifier for the tool.
-  - **`description`**: A human-readable explanation of the tool's purpose.
+  - **`description`**: A human-readable explanation of the tool's purpose. This field will be used by both humans and agents.
 
 - **Toolkit Information:**
 
+[//]: # (On the request side, could the client specify a toolkit ID/name to use?)
```

Comment:
Yes, when calling a tool the client specifies the toolkit's ID as part of the `id` of the tool to execute.